pkg/storaged/anaconda.jsx

```diff
@@ -96,7 +134,21 @@
     const mpm = { };
     for (const p in client.blocks) {
         const b = client.blocks[p];
-        mpm[decode_filename(b.Device)] = block_info(b);
+        mpm[decode_filename(b.PreferredDevice)] = block_info(b);
+    }
+
+    // Add inactive logical volumes
+    for (const vg_p in client.vgroups) {
+        const vg = client.vgroups[vg_p];
+        for (const lv of client.vgroups_lvols[vg_p] || []) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19951-20240206-142939-b945d329-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)